frame/whitelist/src/lib.rs

```diff
@@ -114,12 +114,12 @@ pub mod pallet {
 	}
 
 	#[pallet::storage]
-	pub type WhitelistedCall<T: Config> = StorageMap<_, Twox64Concat, T::Hash, (), OptionQuery>;
+	pub type WhitelistedCall<T: Config> = StorageMap<_, Twox64Concat, Hash, (), OptionQuery>;
```

Comment:
I think we always use `Hash = H256` but it would need a migration if some chains use something different.

Reply:
I very much doubt any are.